packages/groqd/src/types/groq-expressions.test.ts

```diff
@@ -201,3 +201,16 @@ describe("Expressions", () => {
     });
   });
 });
+describe("Expressions.Conditional", () => {
```

Comment:
I don't know if the reason why the test is failing is because there's no `it(...` line here, so it doesn't realize there's a test here? But that seems to be why that suite is failing, because you have a test written out here.

Reply:
Thank you for catching this! Fixed!